feconf.py

```diff
@@ -130,7 +130,7 @@
 ENTITY_TYPE_STORY = 'story'
 ENTITY_TYPE_SUBTOPIC = 'subtopic'
 ENTITY_TYPE_QUESTION = 'question'
-
+ENTITY_TYPE_VOICEOVER_APPLICATION = 'voiceover_application'
 # The maximum number of activities allowed in the playlist of the learner. This
```

Comment:
Add a newline above this

Reply:
Done!

Reply:
I don't see it (the line is still showing as deleted). Add a newline between all the entity types and the next comment

Reply:
![image](https://user-images.githubusercontent.com/16653571/68826663-d4193580-06c4-11ea-8ad5-94ec652fb7c1.png)

For me it doesn't show any deleted line, can you please re-check? (Or maybe I'm misunderstanding your comment?)

Reply:
Oh I just rechecked and it seems to be resolved. Sorry for the confusion... Not sure what happened here!